<launch>
    <node pkg="external_parameter_adjustment" type="parameter_adjustment_server" name="parameter_adjustment_lidar1" output="screen">
        <param name="original_lidar_topic" type="string" value="/front_left/rslidar_points"  />
        <param name="transformation_lidar_topic" type="string" value="/calibration/lidar1"  />
    </node>

    <node pkg="external_parameter_adjustment" type="parameter_adjustment_server" name="parameter_adjustment_lidar2" output="screen">
        <param name="original_lidar_topic" type="string" value="/front_right/rslidar_points"  />
        <param name="transformation_lidar_topic" type="string" value="/calibration/lidar2"  />
    </node>

    <node pkg="external_parameter_adjustment" type="parameter_adjustment_server" name="parameter_adjustment_lidar3" output="screen">
        <param name="original_lidar_topic" type="string" value="/lslidar_point_cloud"  />
        <param name="transformation_lidar_topic" type="string" value="/calibration/lidar3"  />
    </node>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find external_parameter_adjustment)/rviz/external_parameter_adjustment.rviz" />

</launch>
